nodewatcher/modules/devices/migrations/0002_change_cpe_id.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-09-11 12:59
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from nodewatcher.core.generator.cgm import device_migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cgm', '0016_auto_20160206_2343'),
```

Comment:
Since this is manually made,I dont know what should dependencies be set to

Reply:
You should **never** make migrations manually. Use the following command to create an empty migration instead (after removing the manually created migration):
```
python manage.py makemigrations devices --empty -n change_cpe_id
```

And then edit the created file.

Reply:
Also see [Django documentation on migrations](https://docs.djangoproject.com/en/1.10/topics/migrations/).

Reply:
Well,I tried to make them but Django refused to detect changes.
I did not know about empty flag.

I have read documentation on migrations already

Reply:
Yes, you need to use the `--empty` flag when creating data migrations (as opposed to schema migrations, where changes are detected automatically).

Reply:
Thanks for explanation